snmp/tests/conftest.py

```diff
@@ -26,7 +26,7 @@
 )
 
 FILES = [
-    "https://ddintegrations.blob.core.windows.net/snmp/3850.snmprec",
+    "https://ddintegrations.blob.core.windows.net/snmp/cisco-3850.snmprec",
```

Comment:
TODO: Delete `https://ddintegrations.blob.core.windows.net/snmp/3850.snmprec` from Azure once this PR is merged.